internal/environment/command_create.go

```diff
@@ -65,5 +65,5 @@ func (c *command) create(cmd *cobra.Command, args []string) error {
 
 func (c *command) addStreamGovernancePackageFlag(cmd *cobra.Command, defaultValue string) {
 	values := utils.ArrayToCommaDelimitedString([]string{"essentials", "advanced"}, "or")
-	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf("Specify the Stream Governance package as %s.", values))
+	cmd.Flags().String("governance-package", defaultValue, fmt.Sprintf(`Specify the Stream Governance package as %s. Downgrading the package from "advanced" to "essentials" is not allowed.`, values))
```

Comment:
i think this is a bit nuanced. we only don't allow downgrades if the SR cluster is already provisioned. we can probably update it to:
`Downgrading the package from "advanced" to "essentials" is not allowed once the schema registry cluster is provisioned.`
or remove it altogether. will defer to @brianstrauch to decide which is better.